tests/flow.html

```diff
@@ -99,8 +102,15 @@ <h3>
       srcLink.innerText = `VexFlow Source [${ver}]`;
 
       let loadVexFlow;
-      if (isVersionFourOrNewer) {
-        // When loading version >= 4.0.0, only load vexflow-debug-with-tests.js
+      if (useESM) {
+        // This ESM branch only works if flow.html is accessed from a web server
+        // (e.g., `npx http-server` and browse to http://127.0.0.1:8080/tests/flow.html).
+        loadVexFlow = async () => {
+          const module = await import('../build/esm/entry/vexflow-debug-with-tests.js');
+          window.Vex = module.Vex;
+        };
```

Comment:
Adding support for ESM makes debugging circular dependencies a bit easier. Now, each file is loaded separately via import.